app/lib/three_scale/redis_config.rb

```diff
@@ -23,11 +34,7 @@ def reverse_merge!(other)
     end
 
     def method_missing(method_sym, *args, &block)
-      if config.respond_to?(method_sym)
-        config.send(method_sym, *args, &block)
-      else
-        super
-      end
+      config.send(method_sym, *args, &block) if config.respond_to?(method_sym)
```

Comment:
I think we should keep `else super`, otherwise a call to non-existing method will just return `nil` which could be super hard to debug when it creates a problem.